conf/nginx/nginx.conf

```diff
@@ -0,0 +1,32 @@
+error_log stderr debug;
```

Comment:
I think we might want to bump this to `info` level before merging this PR / releasing the next version.